### PR TITLE
CDPCAM-7: Passive Validations for Azure

### PR DESCRIPTION
**Pull request by wonderslug**
This adds a base set of passive validations for Azure.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -32,10 +34,11 @@ repos:
     rev: v2.7.0
     hooks:
       - id: validate_manifest
-  - repo: https://github.com/pre-commit/mirrors-autopep8
-    rev: v1.5.4
-    hooks:
-      - id: autopep8
+  # - repo: https://github.com/pre-commit/mirrors-autopep8
```
Comment (mahendra-rk) [inline .pre-commit-config.yaml:37]: Should we clean this up ?

**Review by mahendra-rk — APPROVED** [review]
LGTM!

File: .pre-commit-config.yaml (hunk 1 above)
Comment (wonderslug) [reply .pre-commit-config.yaml]: We were getting some errors with the generated code.  We can look at it in dev.

**Review by aakanser — APPROVED** [review]
thanks for the merge!

**Pull request merged**